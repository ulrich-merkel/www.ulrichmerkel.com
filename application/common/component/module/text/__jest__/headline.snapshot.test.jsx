/* eslint-disable import/no-extraneous-dependencies, func-names */
import React from 'react';
import renderer from 'react-test-renderer';
import ModuleTextHeadline from './../headline';

describe('common/component/module/text/headline', function () {
    const defaultProps = {
        text: 'Module text headline'
    };

    it('should render correctly', function () {
        const tree = renderer.create(
            <ModuleTextHeadline
                {...defaultProps}
            />
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });
    it('should return null if text is empty', function () {
        const tree = renderer.create(
            <ModuleTextHeadline
                {...defaultProps}
                text={null}
            />
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
