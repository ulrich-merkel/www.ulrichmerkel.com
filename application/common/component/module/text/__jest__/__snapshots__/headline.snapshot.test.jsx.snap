// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`common/component/module/text/headline should render correctly 1`] = `
<h3
  className="is-centered c-type--h3 m-text__headline"
>
  Module text headline
</h3>
`;

exports[`common/component/module/text/headline should return null if text is empty 1`] = `null`;
